### PR TITLE
doc: allow developer to skip examples build

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull-request fixes https://github.com/ansys/ansys-sphinx-theme/issues/526 when the ``BUILD_EXAMPLES=true`` environment variable is set.

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -243,26 +253,33 @@ def download_and_process_files(example_links: List[str]) -> List[str]:
...
+    )
+
+    admonitions_links = download_and_process_files(admonitions_links)
+    todo_include_todos = True  # admonition todo needs this to be True
+
+    jinja_contexts["examples"] = {"inputs_examples": file_names}
+    jinja_contexts["admonitions"] = {"inputs_admonitions": admonitions_links}
+
+
+print(f"JINJA CONTEXTS: {jinja_contexts}")
```
Comment (Revathyvenugopal162) [inline doc/source/conf.py:285]:
```suggestion
```
do we need to print this?

**Review by Revathyvenugopal162 — APPROVED** [review]

File: doc/source/conf.py (hunk 1 above)
Comment (jorgepiloto) [reply doc/source/conf.py:285]: Good catch! Fixing it.

**Comment by jorgepiloto** [comment]
I can't reproduce locally the error raised by the CI/CD pipelines...

**Review by SMoraisAnsys — COMMENTED** [review]
@jorgepiloto I think the error is related to the last section title, it should be "## Render a data frame" instead of "# Render a data frame".

**Comment by jorgepiloto** [comment]
> @jorgepiloto I think the error is related to the last section title, it should be "## Render a data frame" instead of "# Render a data frame".

Thanks, @SMoraisAnsys. This was not the point but you guided me in the right direction. The problem was with the multi-string for the prologue. It is fixed in https://github.com/ansys/ansys-sphinx-theme/pull/553/commits/1fc76c9ee59f93e6d213138d4b6d71d9e118b347

**Pull request merged**